content/en/docs/reference/kubernetes-api/labels-annotations-taints.md

```diff
@@ -57,10 +57,25 @@ This will be set only if you are using a `cloudprovider`. This setting is handy
 if you want to target certain workloads to certain instance types, but typically you want
 to rely on the Kubernetes scheduler to perform resource-based scheduling. You should aim to schedule based on properties rather than on instance types (for example: require a GPU, instead of requiring a `g2.2xlarge`).
 
+**NOTE**: starting in v1.17, this label is deprecated in favor of [node.kubernetes.io/instance-type](nodekubernetesioinstance-type).
```

Comment:
I recommend using a [`{{< note >}}` shortcode](https://kubernetes.io/docs/contribute/style/style-guide/#note) here, and again below. Also the hyperlink is broken.

Eg:
```suggestion
{{< note >}}This label is deprecated in favor of [node.kubernetes.io/instance-type](#nodekubernetesioinstance-type){{< /note >}}
```